spec/card_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
+require 'support/factory_girl.rb'
 
 describe Card do
   before do
-    @card = Card.create(original_text: "How are you?", translated_text: "Kamusta?", review_date: DateTime.now)
+    @card = create(:card)
```

Comment:
@davetoxa is there a particular advantage of using let! vs normal initialization? since let! is not lazily evaluated it feels that resource use is still the same vs normal initialization